packages/google-maps/index.android.ts

```diff
@@ -1971,6 +1975,11 @@ export class Cap implements ICap {
 export class ButtCap extends Cap {
 	#native: com.google.android.gms.maps.model.ButtCap;
 
+	constructor() {
+		super();
+		this.#native = new com.google.android.gms.maps.model.ButtCap();
```

Comment:
The constructors should be allowed take an optional native instance so using `fromNative` doesn't waste time creating a native object for v8 & jvm to track :) . This doesn't need to be added to the public api

Reply:
At this point, wouldn't it make more sense to just add another static method that creates one from scratch and make the constructor private?

Cap.fromNative(nativeObj);

Cap.new() or Cap.empty().

Tbh, I just hate this whole `#private` variable thing too, but I digress haha